astropy/units/equivalencies.py

```diff
@@ -561,7 +562,7 @@ def brightness_temperature(frequency, beam_area=None):
         warnings.warn("The inputs to `brightness_temperature` have changed. "
                       "Frequency is now the first input, and angular area "
                       "is the second, optional input.",
-                      DeprecationWarning)
+                      AstropyDeprecationWarning)
```

Comment:
I don't understand why we did not use `AstropyDeprecationWarning` back in #5173 , so I changed it here since the milestone matches anyway. If you want this separated out as a new PR, please let me know.

Reply:
I vote for fixing it while you are at it!

Reply:
Yes, let's just fix it here.